src/scripts/main.js

```diff
@@ -1,7 +1,37 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const messageWindow = document.createElement('div');
+  const messageTitle = document.createElement('h2');
+  const messageText = document.createElement('p');
+
+  messageTitle.textContent = title;
+  messageTitle.classList.add('title');
+  messageText.textContent = description;
+
+  messageWindow.classList.add('notification');
+  messageWindow.style.top = posTop + 'px';
+  messageWindow.style.right = posRight + 'px';
+
+  if (type === 'success') {
+    messageWindow.classList.add('success');
+  }
+
+  if (type === 'warning') {
+    messageWindow.classList.add('warning');
+  }
+
+  if (type === 'error') {
+    messageWindow.classList.add('error');
+  }
+
+  messageWindow.append(messageTitle);
+  messageWindow.append(messageText);
+  document.body.append(messageWindow);
+
+  setTimeout(() => {
+    messageWindow.style.display = 'none';
+  }, '2000');
```

Comment:
The delay in `setTimeout` should be a number, not a string. Change `'2000'` to `2000` to ensure the timeout works correctly.